QgisModelBaker/tests/utils.py

```diff
@@ -4,6 +4,6 @@
 
 
 @pytest.mark.skip("This is a utility function, not a test function")
-def testdata_path(path):
+def _testdata_path(path):
```

Comment:
Could it be something else than an `_`, since for me it's an indicator for "private" functions.  What about `path_testdata` ? As far as I understand it just must not start with "test", right?

Reply:
Correct, addressed in https://github.com/opengisch/QgisModelBaker/pull/922/commits/085c0293d40099533464742b78a81caa492f3b11